react/components/Onboarding/components/Channels/components/IntroduceChannel/index.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Text from 'react/components/UI/Text';
+
+const IntroduceChannelWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  max-width: 33em;
+  padding: 1em 1em 0 1em;
+  position: relative;
+`;
+
+const BodyWrapper = styled.div`
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  overflow-y: auto;
+  padding: 2em;
+`;
+
+const Section = styled.div`
+  align-self: center;
+  marging: 1em 0;
+`;
+
+const IntroductionText = styled(Text).attrs({
+  f: 7,
+})`
+  text-align: center;
+`;
+
+
+class IntroduceChannel extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.imageSrc = '/images/onboarding/addblock.gif';
```

Comment:
This seems odd to me? Assets should be fingerprinted in production so this may not work there. You could do a require (the webpack config for that might need to be updated, not certain).

Reply:
Yeah. Looks like all of ya'lls images are cloudfront URLs (like [this](https://github.com/aredotna/ervell/blob/master/apps/about/templates/faqs.jade#L45)). I wasnt really sure if you wanted to host the image somewhere and give me the URL or what. I can look at webpack config to require.